packages/frontend-shared/README.md

```diff
@@ -18,26 +22,143 @@ For the best development experience, you will want to use VS Code with the [Vola
+
+If you are familiar with the Options API, which was the main way to write component script sections in Vue 2, the separation of variables and functions into named parts of the Options object like `computed` and `data` provided a familiar, but unwieldy, structure in each component. The Composition API lets us use those features anywhere we like, without dividing things into a predefined structure. `<script setup>` is a way to write Composition API code with less boilerplate and some other advantages described in the docs.
+#### [Pinia](https://pinia.vuejs.org/)
+TS-friendly, modularized state management - this is what we use instead of Vuex for the small amount of global state we have.
+
+#### [VueUse](https://vueuse.org/)
+Broad collection of composable utilities that provides reactive values for various common events and DOM properties, CSS rules, local storage and a lot of other stuff. Very useful if we feel the need to add and tear down low level event listeners, as this library already covers so many and exposes them to Vue correctly. It's like a front-end lodash, so where the VueUse implementation of something works for us, we prefer to use it instead of roll our own.
+
+#### [Headless UI](https://headlessui.dev/)
+We are using some components from Headless UI as the basis for UI patterns like modals, custom dropdowns, and expanding panels. We use Headless UI because it is well documented and the accessibility features are properly thought out. These advantages outweigh the occasional workarounds we have to use in order to get sophisticated behavior working that Headless UI does not support.
+
+### Router
+
+Only `@packages/app` has a router, so details are described in its [README](../app/README.md).
+### Styles
+
+#### Tailwind and Windi
+We use [Tailwind](https://tailwindcss.com/) through [WindiCSS](https://windicss.org/). The codebase is utility-driven and all CSS that can be achieved through utility classes is written that way. The main way to reuse CSS in multiple places is to extract a component that applies the utility classes and can wrap other elements as needed.
+
+#### Explicit Pixel Values
+WindiCSS can create CSS classes as build time based on what class names we use in our components. That means syntax like this will work:
+
+`<p class="p-20px">`
+
+This allows us to specify explicit pixel values for measurements. We follow this pattern throughout the Cypress App codebase.
+
+### Icons
+
+The icons will temporarily live in this package and will soon be moved to `packages/frontend-shared`.
+
+## Custom Icon Library
+Cy has a very custom icon library, to meet the following needs:
+
+* Most of our icons are duo-tone
+* They must be styled in both dark and light contexts (e.g. on a dark menu bar vs on a light background).
+* Since they're duotone, you want to target the specific strokes and fills of the SVGs to color them.
+* All of the niceties of our utility classes should work (e.g. `group-hover` or `group-focus`).
```

Comment:
I don't know what 'niceties' are in this context - what is this bullet point asking me to do?

Reply:
Will reworks these a little - it's nothing the developer is being asked to do, but it does explain why we jump through some hoops and can't just export SVGs and use them as they come from Figma.